README.md

```diff
@@ -61,7 +61,15 @@ __Please note that you must use [express-session](https://github.com/expressjs/s
 * `cookie.options` Object - Optional. A valid Express cookie options object.
 * `angular` Boolean - Optional. Shorthand setting to set `lusca` up to use the default settings for CSRF validation according to the [AngularJS docs]. Can be used with `cookie.options`.
 * `blocklist` Array or String - Optional. Allows defining a set of routes that will not have csrf protection.  All others will.
-* `allowlist` Array or String - Optional. Allows defining a set of routes that will have csrf protection.  All others will not.
+Following configurations are valid:
+  ```
+  blocklist: '/details';
+  blocklist: ['/details', '/summary'];
+  blocklist: [{'/details', match: 'exact'}, '/summary']
```

Comment:
We should also support { path: '/summary', match: 'includes' } which behaves similar to the default behavior.

Plus
```
<  blocklist: [{'/details', match: 'exact'}, '/summary']
>  blocklist: [{ path: '/details', match: 'exact'}, '/summary']
```